source/channels/rename-channels.rst

```diff
@@ -5,10 +5,27 @@ Rename channels
   :start-after: :nosearch:
 
 Anyone can rename the channels they belong to, unless the System Admin has `restricted the permissions to do so using advanced permissions </onboard/advanced-permissions.html>`__.
+.. tabs::
 
-In the desktop app, select the channel name at the top of the center pane to access the drop-down menu, then select **Rename Channel**. You'll be prompted for two values:
+  .. tab:: Web browser/Desktop app
 
-- **Channel display name:** This appears in the Mattermost user interface. 
-- **Channel handle:** This is part of the channel URL. You can also change the channel handle when renaming a channel, but changing channel handles may break existing links.
+      **To Rename  channel in the desktop app**
+
+        In the desktop app, select the channel name at the top of the center pane to access the drop-down menu, then select **Rename Channel**. You'll be prompted for two values:
+
+        - **Channel display name:** This appears in the Mattermost user interface. 
+        - **Channel handle:** This is part of the channel URL. You can also change the channel handle when renaming a channel, but changing channel handles may break existing links.
+
+        For example, for the following channel, ``https://community.mattermost.com/core/channels/ux-design``, The channel display name would be ``UX Design``, and the channel handle would be ``ux-design``.
+
+    .. tab:: Mobile app
+       
+        **To Rename  channel in the mobile app V2**
+
+           In the mobile app, select the channel name from the sidebar to access the menu, then select **Edit Channel** option. There will be three values:
+           - **Name:** you can give a suitable name which will appear in the Mattermost user interface.
+           - **Purpose:** you can write the motive of the channel.
+           - **Header:** we can use it to list various links and repos.
+
+           For example, for the following channel, ``https://community.mattermost.com/core/channels/ux-design``, The channel display **Name** would be ``UX Design``,             **purpose** is to discuss about UX by core contributors and staff, **Header** is the links and repos listed under Header option.
```

Comment:
I don't think this is relevant here as the bullet points should cover the use cases sufficiently. And we don't want to be prescriptive about optional fields.

Reply:
then should I just keep Name option and remove purpose and header?
@justinegeffen

Reply:
I was referring to the sentence describing the URL and so on. The three values should remain. :)